src/nft/components/card/containers.tsx

```diff
@@ -0,0 +1,850 @@
+  ProfileSelectedAssetIcon,
+  SudoSwapIcon,
+  TagIcon,
+  X2y2Icon,
+} from 'nft/components/icons'
+import { Markets, UniformAspectRatio, UniformAspectRatios } from 'nft/types'
+import { putCommas } from 'nft/utils'
+import { ReactNode, useEffect, useRef, useState } from 'react'
+import { AlertTriangle, Pause, Play } from 'react-feather'
+import { Link } from 'react-router-dom'
+import { useDarkModeManager } from 'state/user/hooks'
+import styled from 'styled-components/macro'
+import { BREAKPOINTS, ThemedText } from 'theme'
+import { colors } from 'theme/colors'
+
+const BORDER_RADIUS = '12'
+
+const SuspiciousIcon = styled(AlertTriangle)`
+  width: 16px;
+  height: 16px;
+  color: ${({ theme }) => theme.accentFailure};
+`
+
+const StyledImageContainer = styled.div<{ isDisabled?: boolean }>`
+  position: relative;
+  pointer-events: auto;
+  &:hover {
+    opacity: ${({ isDisabled, theme }) => (isDisabled ? theme.opacity.disabled : theme.opacity.enabled)};
+  }
+  cursor: ${({ isDisabled }) => (isDisabled ? 'default' : 'pointer')};
+`
+
+const StyledDetailsRelativeContainer = styled.div`
+  position: relative;
+  height: 84px;
+`
+
+const StyledDetailsContainer = styled(Column)`
+  position: absolute;
+  width: 100%;
+  padding: 16px 8px 0px;
+  justify-content: space-between;
+  gap: 8px;
+  height: 84px;
+  background: ${({ theme }) => theme.backgroundSurface};
+  will-change: transform;
+  transition: ${({ theme }) => `${theme.transition.duration.medium} ${theme.transition.timing.ease} transform`};
+
+  @media screen and (max-width: ${BREAKPOINTS.sm}px) {
+    height: 112px;
+    transform: translateY(-28px);
+  }
+`
+
+const StyledActionButton = styled(ThemedText.BodySmall)<{
+  selected: boolean
+  isDisabled: boolean
+}>`
+  position: absolute;
+  display: flex;
+  padding: 8px 0px;
+  bottom: -32px;
+  left: 8px;
+  right: 8px;
+  color: ${({ theme, isDisabled }) => (isDisabled ? theme.textPrimary : theme.accentTextLightPrimary)};
+  background: ${({ theme, selected, isDisabled }) =>
+    selected ? theme.accentCritical : isDisabled ? theme.backgroundInteractive : theme.accentAction};
+  transition: ${({ theme }) =>
+    `${theme.transition.duration.medium} ${theme.transition.timing.ease} bottom, ${theme.transition.duration.medium} ${theme.transition.timing.ease} visibility`};
+  will-change: transform;
+  border-radius: 8px;
+  justify-content: center;
+  font-weight: 600 !important;
+  line-height: 16px;
+  visibility: hidden;
+  cursor: ${({ isDisabled }) => (isDisabled ? 'default' : 'pointer')};
+
+  @media screen and (max-width: ${BREAKPOINTS.sm}px) {
+    visibility: visible;
+    bottom: 8px;
+  }
+
+  &:before {
+    background-size: 100%;
+    border-radius: inherit;
+
+    position: absolute;
+    top: 0;
+    left: 0;
+
+    width: 100%;
+    height: 100%;
+    content: '';
+  }
+
+  &:hover:before {
+    background-color: ${({ theme, isDisabled }) => !isDisabled && theme.stateOverlayHover};
+  }
+
+  &:active:before {
```

Comment:
Not noticing any difference when hovering the button on dark mode (can see a slight change on light mode)
![buttonHover](https://user-images.githubusercontent.com/11512321/224792730-e389758c-c1dd-40b8-bcce-7b7e5f53713f.gif)

Reply:
there is actually a super slight change - that is the way its implemented on swap page too